nifi-nar-bundles/nifi-provenance-repository-bundle/nifi-persistent-provenance-repository/src/test/java/org/apache/nifi/provenance/TestPersistentProvenanceRepository.java

```diff
@@ -1914,112 +1916,6 @@ public void testFailureToCreateWriterDoesNotPreventSubsequentRollover() throws I
     }
 
 
-    @Test
-    public void testBehaviorOnOutOfMemory() throws IOException, InterruptedException {
```

Comment:
This test was assuming that the Record Writer was the SimpleRecordWriter and really made some other assumptions that it shouldn't make. It wasn't a well-written unit test, in general, as it was assuming dependencies between classes, so was really more of an integration test. I considered refactoring to make it work but then decided to simply remove it, since we already have a JIRA anyway for fixing how NiFi handles OOME by letting the process die and having the bootstrap restart it.